x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/200_query_ruleset_put.yml

```diff
@@ -0,0 +1,102 @@
+
+
+---
+'Create Query Ruleset':
```

Comment:
Note: Since only the `PUT` API call is implemented and not `GET` or `DELETE`, this test is pretty lightweight. It will be extended as additional CRUD API calls are added.